gadget/install/install.go

```diff
@@ -53,6 +54,11 @@ func deviceFromRole(lv *gadget.LaidOutVolume, role string) (device string, err e
 // Run bootstraps the partitions of a device, by either creating
 // missing ones or recreating installed ones.
 func Run(gadgetRoot, device string, options Options, observer gadget.ContentObserver) (*InstalledSystemSideData, error) {
+	logger.Noticef("installing a new system")
+	logger.Noticef("        gadget data from: %v", gadgetRoot)
+	if options.Encrypt {
+		logger.Noticef("        encryption: on")
```

Comment:
Nitpick, this would be fine as a single line, without encrypt conditional.
```
logger.Noticef("installing a new system (gadget data: %v", encryption: %v)", gadgetRoot, options.Encrypt)
```